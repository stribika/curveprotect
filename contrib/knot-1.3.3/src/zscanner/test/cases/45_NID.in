$ORIGIN	.
$TTL	1

; The NID is the same as the L64, so there are the differences and basics only.

; OK
@	NID	0	0000:0000:0000:0000	; The simplest case
@	NID	\# 10 00000000000000000000	; Hexadecimal rdata
@	TYPE104	\# 10 00000000000000000000	; TYPE + Hexadecimal rdata
@	TYPE104	0	0000:0000:0000:0000	; TYPE
@	nid	0	0000:0000:0000:0000	; Type in lower-case

; KO
@	NID
