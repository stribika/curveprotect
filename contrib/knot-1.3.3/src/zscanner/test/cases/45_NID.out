OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0068
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0068
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0068
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0068
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0068
RDATA=00000000000000000000
------
WARNG=ZSCANNER_EBAD_NUMBER
------
